src/struct_named_trajectory.jl

```diff
@@ -287,6 +292,7 @@ function NamedTrajectory(
         Z.final,
         Z.goal,
         Z.components,
+        z.params,
```

Comment:
looks like this just needs to be upper case

Reply:
yeah, already fixed